servicetalk-concurrent-test-internal/src/main/java/io/servicetalk/concurrent/test/internal/AwaitUtils.java

```diff
@@ -30,10 +31,25 @@ private AwaitUtils() {
         // no instances
     }
 
+    /**
+     * Await a {@link CountDownLatch} until the latch is counted down, throws unchecked InterruptedException.
```

Comment:
Since the exception now is unchecked I could not include it in the method signature. This means that Javadoc cannot have `@throws`, otherwise, the build fails. I thought this could be a way to specify that the method now throws an unchecked exception, but I am not 100% sure.

Reply:
Since this class is in "internal" package, don't worry about documentation. Let's leave it without `throws` in javadoc.